Configuration/Geometry/g4n.log

```diff
@@ -0,0 +1,7 @@
+----- Begin Fatal Exception 24-Sep-2020 19:32:24 CEST-----------------------
```

Comment:
Can this file be removed?

Reply:
Sorry - I missed it. I did a "git add ." without realizing the consequence

Reply:
Could the two commits be squashed together to avoid the file appearing in the git history (even if it is small)?